packages/core/src/state-management/anchor-resuming-service.ts

```diff
@@ -4,8 +4,9 @@ import type { Repository } from './repository.js'
 import { LogStyle, type DiagnosticsLogger } from '@ceramicnetwork/common'
 import { TaskQueue } from '../ancillary/task-queue.js'
 
-const RESUME_QUEUE_CONCURRENCY = 30
+const RESUME_QUEUE_CONCURRENCY = 5
 const RESUME_BATCH_SIZE = RESUME_QUEUE_CONCURRENCY * 5
+const RATE_LIMIT_DELAY = 100
```

Comment:
with .1 sec delay between streams we can resume 10 streams a second.  That's 600 streams a minute or 36000 streams an hour.  I think that should be plenty fast enough, at least for now until we can optimize how the polling works.  No one currently is going to have 36k streams with pending anchors at once, and taking an hour to resume them is actually fine since CACAOs expire on an order of 24 hours and we strive to anchor any stream within 12 hours.  A 1 hour delay on detecting an anchor commit, while not ideal as it can lead to data forks, is still unlikely to cause data corruption and so is probably acceptable for now.

We should definitely consider increasing the concurrency and removing the rate limit if we get to a world where this only a single background task that polls for all pending anchors at once.